src/app/TideMatchSet.cpp

```diff
@@ -306,6 +307,153 @@ void TideMatchSet::report(
               compute_sp ? &sp_map : NULL, rwlock);
 }
 
+
+/// added by Yang
+void TideMatchSet::writeHeadersDIA(ofstream* file, bool sp) {
```

Comment:
I see a "bool compute_sp" elsewhere in this file. Maybe rename "sp" to "compute_sp" so it is consistent across file?